### PR TITLE
Fixes beeboxes not producing workers

### LINKED ISSUE #32609: bee boxes dont make their free worker bee, meaning they can't do shit
see title

### PR DESCRIPTION
**Pull request by Cyberboss**
Fixes #32609 

:cl: 
fix: Fixed apiaries not producing worker bees
/:cl:

### CODE REVIEW

**Comment by RogueSteampunker** [comment]
B E E S

File: code/modules/hydroponics/beekeeping/beebox.dm [hunk 1]
```diff
@@ -97,7 +97,7 @@
 
 		if(bees.len < get_max_bees())
 			var/freebee = FALSE //a freebee, geddit?, hahaha HAHAHAHA
-			if(bees.len <= 1) //there's always one set of worker bees, this isn't colony collapse disorder its 2d spessmen
+			if(bees.len <= 2 //there's always one set of worker bees + the queen, this isn't colony collapse disorder its 2d spessmen
```
Comment (vuonojenmustaturska) [inline code/modules/hydroponics/beekeeping/beebox.dm]: A new @kevinz000 rises

**Comment by Iamgoofball** [comment]
my entire traitor gimmick fell apart due to this fucking bug

**Comment by Armhulen** [comment]
//there's always one set of worker bees + the queen, this isn't colony collapse disorder its 2d spessmen

haha it actually did do that though

**Comment by Doppeltongue** [comment]
Goof, when you did it, did you not leave any grown plants near the hive? When I tried using bees, one box happened to be 3 tiles away from a fully grown plant and produced bees, the other was about 6 tiles away from the nearest grown plant and didn't produce bees until I planted something 3 tiles away and it grew ready for harvest.

File: code/modules/hydroponics/beekeeping/beebox.dm [hunk 2]
```diff
@@ -97,7 +97,7 @@
 
 		if(bees.len < get_max_bees())
 			var/freebee = FALSE //a freebee, geddit?, hahaha HAHAHAHA
-			if(bees.len <= 1) //there's always one set of worker bees, this isn't colony collapse disorder its 2d spessmen
+			if(bees.len <= 2) //there's always one set of worker bees + the queen, this isn't colony collapse disorder its 2d spessmen
```
Comment (RemieRichards) [inline code/modules/hydroponics/beekeeping/beebox.dm]: Why does this fix it.
A constructed apiary would have only the queen, making bees.len 1, which is covered by the ```<=```, so it produces 1 set of worker bees, this takes it to 2 bees total, the queen and 1 set of workers.

With your change, they'll surely get two free sets of workers now?

Comment (Cyberboss) [reply code/modules/hydroponics/beekeeping/beebox.dm]: hmm

Comment (Iamgoofball) [reply code/modules/hydroponics/beekeeping/beebox.dm]: remie they aren't making ANY BEES AT ALL right now

Comment (Bawhoppen) [reply code/modules/hydroponics/beekeeping/beebox.dm]: Did you test it? Also which plants did you use?

They worked recently

Comment (ShizCalev) [reply code/modules/hydroponics/beekeeping/beebox.dm]: Premade worked when I was testing #32523

Comment (Bawhoppen) [reply code/modules/hydroponics/beekeeping/beebox.dm]: Goof probably didn't even have any plants around or something

Comment (RemieRichards) [reply code/modules/hydroponics/beekeeping/beebox.dm]: it's got nothing to do with plants.
I bet he didn't put any honeyframes in, meaning the hive couldn't support any bees, so ```if(bees.len < get_max_bees())``` would have been 1 < 0, which is false, so no bee generation at all.

Comment (Iamgoofball) [reply code/modules/hydroponics/beekeeping/beebox.dm]: I had 3 fucking frames in tne box

Comment (Iamgoofball) [reply code/modules/hydroponics/beekeeping/beebox.dm]: >premade

Did you test the cargo crate one?

**Pull request closed without merging**